lib/extensions/ar_region.rb

```diff
@@ -117,6 +117,18 @@ def partition_objs_by_remote_region(objs)
     def partition_ids_by_remote_region(ids)
       ids.partition { |id| self.id_in_current_region?(id) }
     end
+
+    private
+
+    def discover_my_region_number
+      region_file = File.join(Rails.root, "REGION")
```

Comment:
Minor...You can just do `Rails.root.join("REGION")`.  No need to update for this PR unless you happen to need to make more changes.